docusaurus/docs/Android/03-ui/01-overview.mdx

```diff
@@ -39,11 +48,11 @@ Since this library uses Material elements, make sure that you use a Material the
 <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-If you want to keep using an `AppCompat` theme for styling, you can use a [Bridge Theme](https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md#bridge-themes) to support using Material based components at the same time.
+If you want to keep using an `AppCompat` theme for styling, you can inherit from a [Bridge Theme](https://m2.material.io/develop/android/docs/getting-started#bridge-themes) to support using Material based components.
```

Comment:
They don't mention bridge theme anymore anywhere in https://github.com/material-components/material-components-android